pandas/io/excel/_base.py

```diff
@@ -689,14 +689,16 @@ class ExcelWriter(metaclass=abc.ABCMeta):
         host, port, username, password, etc., if using a URL that will
         be parsed by ``fsspec``, e.g., starting "s3://", "gcs://".
 
-        .. versionadded:: 1.2.0
-    if_sheet_exists : {'error', 'new', 'replace'}, default 'error'
+        .. versionchanged:: 1.4.0
+    if_sheet_exists : {'error', 'new', 'replace', 'overlay'}, default 'error'
```

Comment:
I think we still want to keep the `versionadded` and add a `versionchanged` beneath it (not sure what pandas's policy is on this). People might still be using 1.2 and 1.3.

Reply:
I struggled quite a bit getting this to work nicely in ReST. I did some more documenting in the meantime... will try!
I haven't found anything on the Panda's policy on this.

Reply:
the `encoding` parameter of `read_csv` has multiple `versionchanged`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html

I think the trick is to have at least one newline between them.

Reply:
![image](https://user-images.githubusercontent.com/33122845/140537999-6e6439b7-824b-4970-a4f1-eb57821b86df.png)
I now have it like this. For some reason the CSS does not load automatically like in `read_csv`:
![image](https://user-images.githubusercontent.com/33122845/140538083-b039457d-7bcc-4964-8431-d10b6f5a1777.png)
But I guess that's beyond the scope of this PR

Reply:
Also, I now noticed that the notices are below the parameters, in stead of as sort of headers. Fixed that